src/garage/torch/algos/vpg.py

```diff
@@ -261,12 +323,11 @@ def _compute_policy_entropy(self, obs):
 
         return policy_entropy
 
-    def _compute_objective(self, advantages, valids, obs, actions, rewards):
+    def _compute_objective(self, advantages, obs, actions, rewards):
         """Compute objective value.
 
         Args:
             advantages (torch.Tensor): Expected rewards over the actions.
-            valids (list[int]): Array of length of the valid values.
             obs (torch.Tensor): Observation from the environment.
             actions (torch.Tensor): Predicted action.
```

Comment:
if you are going to keep this documentation, please make it correct and complete

Reply:
removed.